content/projects/gsoc/2020/projects/git-plugin-performance.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 layout: gsocproject
 title: "Git Plugin Performance Improvements"
-goal: "Improve end-to-end performance of Git and Git Client Plugins"
+goal: "Enhance user experience by enabling faster checkouts"
```

Comment:
Do you think I should be more specific? 
Something like: Enhancing user experience by improving git plugin performance

Reply:
I think this is fine.

Reply:
A more detailed description would be great, but this one is fine as well